pages/studentInstanceQuestionHomework/studentInstanceQuestionHomework.ejs

```diff
@@ -76,5 +76,7 @@
         </div>
       </div>
     </div>
+    // Needs to be located below element scripts
```

Comment:
```suggestion
    <!-- Needs to be located below element scripts -->
```

Reply:
Aha good catch :slightly_smiling_face:

Reply:
Somehow I didn't notice these until you caught the first one 🤦‍♂️